### DISCUSSION TITLE
[SOLVED] Review on ghacks.net

### DISCUSSION THREAD

**Discussion by umbellipher** [martinrotter/rssguard#567] [General]
### Brief description of the feature request

I see there is a very complimentary review of Rssguard at https://www.ghacks.net/2022/01/03/rss-guard-open-source-feed-reader-with-optional-online-feed-syncing/ this morning.

Well deserved, in my opinion.

**Comment by martinrotter** [comment]
Converting to discussion.

**Comment by martinrotter** [comment]
Wow, let me check how much they dragged the app through shit.

**Comment by umbellipher** [comment]
As you will see, it is highly complimentary. Brinkmann might even see it as a replacement for his beloved QuiteRSS, which is saying something. Kudos to you!

**Reply by martinrotter** [reply]
Well, just finished reading. Yes, I am surprised as I usually, people just worship quiterss and insta-reject anything else. That's a progress, I guess. :) Thanks for heads up.

**Reply by Owyn** [reply]
Cuz it just works from the box :)

I probably would've left rssguard after trying it out if not the discussion where you described ultra-hackering ways to fix it - e.g. QT_SCALE_FACTOR system variable

and most favicons in rssguard don't work :D

**Reply by martinrotter** [reply]
Simply set that env. variable globally and that's it.

> and most favicons in rssguard don't work :D

? Ellaborate.

**Reply by Owyn** [reply]
> Simply set that env. variable globally and that's it.

you know it's not that simply for most users... and they won't even know about it in the first place

> > and most favicons in rssguard don't work :D
> 
> ? Ellaborate.

You've seen it in the solved theme about website scrapping and said you was going to check what's wrong I think...

RSSGuard: (clicking "fetch metadata" doesn't help)
![image](https://user-images.githubusercontent.com/1309656/149766545-f70c50e6-fa5f-46f2-a11d-44609e65aa79.png)

Other rss client:
![image](https://user-images.githubusercontent.com/1309656/149766566-ef832d7e-3853-4894-8aa9-5495a007ff12.png)

Example feed url: https://manga4life.com/rss/Dead-Mount-Death-Play.xml

**Reply by martinrotter** [reply]
let me check

**Reply by martinrotter** [reply]
![image](https://user-images.githubusercontent.com/1255302/149774952-d413dd0b-e494-49b2-8e24-19e9bfbda925.png)

pushed a fix: aa334a90a

If you have any other feeds which are not fetching icons properly, make a list pls.

**Reply by Owyn** [reply]
> If you have any other feeds which are not fetching icons properly, make a list pls.

Here
[rssguard_feeds_2022-01-17.zip](https://github.com/martinrotter/rssguard/files/7882331/rssguard_feeds_2022-01-17.zip)

**Reply by martinrotter** [reply]
Imported, tested, except one icon, all work:

![image](https://user-images.githubusercontent.com/1255302/149785812-644a934b-c5dd-4511-b8da-4712e454eae3.png)

**Reply by Owyn** [reply]
Weird, by going to the icon url - it works there in the browser
![image](https://user-images.githubusercontent.com/1309656/149786471-dd9241c6-f59f-4ed4-b6f2-bf4213d5836d.png)

**Reply by martinrotter** [reply]
Yes, that is because your browser automagically loads icon with S2 google service. I do not use S2 google service in RSS Guard, I use duck duck go icon finder which may give different results for some URLs, although in general it works nicely. That feed sadly does not include direct link to icon/image either, so it is lost cause, other icons should work nice.

**Reply by Owyn** [reply]
What about manhuaus.com website? sadly favicon in RSSguard doesn't work either for it

**Reply by martinrotter** [reply]
should work with latest devbuild:

![image](https://user-images.githubusercontent.com/1255302/150122902-6246752a-dee9-4a65-9fb4-022dba1545da.png)

**Reply by Owyn** [reply]
The new devbuild after that last one or that last one? I'm using that one and no icon:
![изображение](https://user-images.githubusercontent.com/1309656/150123999-cd80e7fd-9a6a-4a23-8b5d-0f2c48630ca5.png)

**Reply by martinrotter** [reply]
what revision -> see about dialog

**Reply by Owyn** [reply]
It is disallowed to copy the revision number 🤣 
![изображение](https://user-images.githubusercontent.com/1309656/150125646-4519a8ab-dac4-45c7-a312-51626afd8f1d.png)

**Reply by martinrotter** [reply]
That's way too old devbuild, download newest ones when it appears.

**Reply by martinrotter** [reply]
RSS Guard
Verze: 4.1.3 (vytvořená Windows/x86_64)
Kontrola: 9c670b862
Datum buildu: 19.01.22 11:24
Qt: 5.15.2 (kompilováno proti 5.15.2

You need to select it first.

**Comment by nhantrn** [comment]
Another Martin too! Congrats Martin!

**Comment by Crehet** [comment]
I agree with the most of the comments favoring QuiteRSS. 'cause it is extremely powerful its the only one that satisfies my needs.
I'd love very very very much if RSS Guard could handle all these, they are for me quite the important features.

---
- Remember preferred `Sort` position (it works per feed and per folder i believe) - **important feature**
- Star Icons - **i'll miss this feature**
![Star Icons](https://user-images.githubusercontent.com/119722248/215221028-469f7cc2-4208-4b06-8b47-8c2a054ca882.png)
- Notification Options **i'll miss this feature**
![Notifications](https://user-images.githubusercontent.com/119722248/215221218-c98183d4-46dd-4f4f-8c07-bfdc9d492c83.png)
    - Notification Window - **important feature**
    ![Notification Window](https://user-images.githubusercontent.com/119722248/215221331-aaaa15ca-5e63-4920-a609-35b453a5cd9a.png)
    - Notification Window - Preferred Feeds - **important feature**
    ![Notification Window - Preferred feeds](https://user-images.githubusercontent.com/119722248/215221325-30211b70-3be1-452e-96b5-3c3d1e6c1af9.png)
- Font&Colors (they also affect the text color from feeds content) - **i'll miss this feature (you can do your own css and whatnot but i dunno about code and prefer GUI)**
![Fonts & Colors](https://user-images.githubusercontent.com/119722248/215221673-68da50fb-3a71-42e1-ba26-17ea734a8291.png)
- Folder Options - **important feature (pretty much the same as *Crucial options on first Tab***
![Folder Options](https://user-images.githubusercontent.com/119722248/215222023-e0392a1f-c9c6-4198-97c3-61f80fb5d53a.png)
- Sorting By Columns - **important feature (folders can also be sorted)**
![Sorting By Colums](https://user-images.githubusercontent.com/119722248/215221339-01d48419-2039-4be1-a087-3c41afc27f89.png)
- Display Options (works for folders too)
![Display Options](https://user-images.githubusercontent.com/119722248/215221335-0170215f-fbab-4099-b863-3989cf0e0573.png)
- Crucial Options on first tab - **important feature (as of current version of this app, not being able to mass modify enable/disable of feeds and modify its update intervals is extremely time consuming**
![Crucial options on first Tab](https://user-images.githubusercontent.com/119722248/215221338-d541d1bd-8514-47f9-a6d8-8736bc08972f.png)
- Filter Rules - **important Feature (mainly for GUI interaction) our dear maintainer has already refused to do it himself though, we are relying on a PR from anyone at this point** 
![Filter Rules](https://user-images.githubusercontent.com/119722248/215224271-25081de5-d08a-4b62-bd0c-e80769ed0bcf.png)

Overall i will just simply miss QuiteRSS.

**Reply by martinrotter** [reply]
It is fair to say that rssguard is NOT quite as and never will have same feature set.

Some of your suggestions are actually on my internal to-do list, and will make it into app one day. Note that I am very busy with my day to day life 

Some of your suggestions will never be in rssguard.

**Reply by Crehet** [reply]
> It is fair to say that rssguard is NOT quite as and never will have same feature set.

yeah, it would otherwise be a rip off of QuiteRSS. and that is a no-kay from a developer point of view.

> Some of your suggestions are actually on my internal to-do list, and will make it into app one day.

don't say _some_, it's scary. i already know which one is not going to happen though, i dunno which is the other one, probably "Crucial Options on first tab", "star icon", and "Font&Colors". a revamp of the notifications feature is likely to happen i believe. and also "Remember sort position". but don't tell me, i'll simply wait for the depression to come.

> Note that I am very busy with my day to day life

keep it up, we counting on you.

> Some of your suggestions will never be in rssguard.

...sob.

your RSS Feeder application is already far from the typical basic RSS Feeder, so yeah, i have good feelings about this project, it will definitely replace QuiteRSS some day.           i hope.

**Reply by martinrotter** [reply]
> > It is fair to say that rssguard is NOT quite as and never will have same feature set.
> 
> yeah, it would otherwise be a rip off of QuiteRSS. and that is a no-kay from a developer point of view.
> 
> > Some of your suggestions are actually on my internal to-do list, and will make it into app one day.
> 
> don't say _some_, it's scary. i already know which one is not going to happen though, i dunno which is the other one, probably "Crucial Options on first tab", "star icon", and "Font&Colors". a revamp of the notifications feature is likely to happen i believe. and also "Remember sort position". but don't tell me, i'll simply wait for the depression to come.
> 
> > Note that I am very busy with my day to day life
> 
> keep it up, we counting on you.
> 
> > Some of your suggestions will never be in rssguard.
> 
> ...sob.
> 
> your RSS Feeder application is already far from the typical basic RSS Feeder, so yeah, i have good feelings about this project, it will definitely replace QuiteRSS some day. i hope.

OK, I and this discussion from my side. If you want to troll or start flamewar, do it somewhere else.

RSS Guard is open-source effort and anyone can submit well-structured PRs and in past some good PRs arrived. Simple as that. I do this for free in my free time which could be very well invested elsewhere. If you want to use RSS Guard, use it. If not, then just move along.

ALSO, submit each of your FRs in separate ticket.

**Reply by Crehet** [reply]
> OK, I and this discussion from my side. If you want to troll or start flamewar, do it somewhere else.

you took my comment quite negatively... i gave a bit of a my opinion and personal taste on what i believed to be the right place (here) instead of doing it on ghacks site, but flamewar... i guess my comment could trigger more people to post their preferences here, by bad.

> RSS Guard is open-source effort and anyone can submit well-structured PRs and in past some good PRs arrived

yeah, i've seen them reason why i voiced my personal feelings for the future of this project.

> I do this for free in my free time which could be very well invested elsewhere.

yeah, i know. i should have phrased my words differently, my bad.

> If you want to use RSS Guard, use it. If not, then just move along.

i love both apps. and i've already made my move to rssguard.

> ALSO, submit each of your FRs in separate ticket. 

this one [883#](https://github.com/martinrotter/rssguard/issues/883#issue-1560425917) no? i've updated it.

**Reply by martinrotter** [reply]
@Crehet Sorry for some of my yesterday's words, I had the worst mood in months. My apology.
